users/migrations/0004_test_data.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations, transaction
+from django.contrib.auth.models import User
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0001_initial'),
+        ('users', '0003_alter_profile_dob'),
+    ]
+
+    def generate_data(apps, schema_editor):
+
+        users_list = [
+            User.objects.create_user('Test-user-profile1', 'Test@gmail.com', 'password777'),
```

Comment:
Consider adding Profie creation to that test, and then use it

Reply:
I thought that the test data should show that the profile is auto generated when creating a new user.
The test data of a profile is given in #59 
Or the test data here should look like the test data in #59 ?

Reply:
What do you plan to test here?
See the test_data content in PR#59?

    username = 'Test-user777'
    email = 'Test@gmail.com'
    password = 'password777'

    username2 = 'Test-user222'
    email2 = 'Test777@gmail.com'
    password2 = 'password777'

profile_test_data = [
            (users_list[0], 'default.jpg', '2010-09-08'),
            (users_list[1], 'default.jpg', '2000-01-01')
        ]

Reply:
I want to test here if a profile is created automatically every time a user is created.
So I create a test user and expect to see the profile with defaults fields:

![image](https://user-images.githubusercontent.com/72739568/115244237-6d574180-a12c-11eb-8214-eb4f4a2cb6fb.png)

Reply:
LGTMT, how do you verify automatically it is created? 
Maybe there is a missing part ? in the test? 

Please refer latest part in last talk testing 
@pytest.mark.django_db
class TestProfile:
    def test_profie_created(cls):